scripts/windows/BuildSetupTemplate.js

```diff
@@ -36,6 +36,7 @@ const valuesToReplace = {
     AppExecutableName: executableName,
     AppSetupExecutableName: `${prodName}-${version}-${buildFolderPrefix}win`,
     Version: version,
+    SetupIconFile: path.join(__dirname, "..", "..", "assets", "images", "oni2.ico"),
```

Comment:
The "Icon"/"DefaultIcon" bits below will also need updating with this new path, since I just put a temp `{app}\\oni2.ico` one in.

Reply:
Thanks @CrossR for reviewing!

I _think_ it should be OK - I added a line to copy the `oni2.ico` to the root application directory (the change above in `scripts/release.js` - so that `{app}\\oni2.cio` should be valid.

Seems like it works (although we'll need a better icon at some point!)
![image](https://user-images.githubusercontent.com/13532591/62082419-98d42300-b209-11e9-85f7-4c6f80962509.png)